ido/src/ft_contract.rs

```diff
@@ -56,17 +68,13 @@ impl IDOContract {
         match env::promise_result(0) {
             PromiseResult::NotReady => unreachable!(),
             PromiseResult::Successful(_value) => {
-                // let upgradable_account: UpgradableAccount = self.accounts.get(&account_id).unwrap();
-                // let mut account: Account = Account::from(upgradable_account);
-
-                // // update account data
-                // account.pre_reward = 0;
-                // account.last_block_balance_change = env::block_index();
-
-                // self.accounts.insert(&account_id, &UpgradableAccount::from(account));
-                // self.total_paid_reward_balance += amount.0;
-
-                // amount
+                let mut project_account_unordered_map = self.internal_get_accounts_by_project_or_panic(project_id);
```

Comment:
We should update the claimed amount in the `project_account ` before starting to transfer token to user.